_DeploymentAndDistroPackaging/ansible/roles/docker/meta/main.yml

```diff
@@ -27,7 +27,9 @@ galaxy_info:
   platforms:
   - name: Ubuntu
     versions:
-      - xenial
+      - Trusty
```

Comment:
This is just informative when the role is uploaded to ansible-galaxy not really "executed" or "parsed" code. I did test on Xenial. and pulling Trusty and Yakkety imaged now. but the instructions are the same from https://docs.docker.com/engine/installation/linux/ubuntu/